<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionHelper</class>
 <widget class="QWidget" name="ConnectionHelper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>connection helper</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditReceiver"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>sender</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>receiver</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>signal</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditSender"/>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>store result in variable</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QComboBox" name="comboBoxReceiverType">
        <item>
         <property name="text">
          <string>SLOT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SIGNAL</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxSignal">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>clicked(bool)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>toggled(bool)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBoxSlot"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
